src/maps/hashmap.js

```diff
@@ -0,0 +1,134 @@
+const IAMap = require('iamap')
+const murmurhash3 = require('murmurhash3js-revisited')
+const Type = require('../base')
+const Block = require('@ipld/block')
+
+const _type = 'IPLD/Experimental/HashMap/0'
```

Comment:
I still can't _not_ version these things .. even with "Experimental"

Reply:
Ya, you’re right, I’m going to go in and update the rest to do the same.